src/styles/product__card.scss

```
@@ -9,39 +9,48 @@
   flex-direction: column;
   width: $card-width;
   height: 403px;
-  padding: 14px;
-  border: 1px solid #f3f3f3;
-  border-radius: $border-radius;
+  padding: 30px 14px 15px;
   margin: 0 $card-margin 50px;
+  border: 1px solid $border-color;
+  border-radius: $border-radius;
 
   &__photo {
-    margin: 16px 5px 41px;
+    width: 160px;
+    height: 134px;
+    align-self: center;
   }
 
   &__img {
     width: 160px;
     height: 134px;
+
+    transition: transform 1s;
+
+    &:hover {
+      outline: 3px solid $border-color;
+      transform: scale(1.2) translateY(-14px);
+    }
   }
 
   &__about {
     font-weight: 500;
     font-size: 12px;
     line-height: 15px;
-    margin: 0 0 5px;
+    margin: 45px 0 0;
```

Якщо ти не обнуляєш бокові та нижній маргіни, то простіше було б

```suggestion
    margin-top: 45px;
```